test/unit/lib/runtimeSettings_spec.js

```diff
@@ -0,0 +1,147 @@
+const should = require('should') // eslint-disable-line
+const runtimeSettings = require('../../../lib/runtimeSettings')
+const fs = require('fs/promises')
+const path = require('path')
+const os = require('os')
+
+describe('Runtime Settings', function () {
+    let TMPDIR
+    before(async function () {
+        TMPDIR = await fs.mkdtemp(path.join(os.tmpdir(), 'fflauncher-'))
+    })
+    after(async function () {
+        await fs.rm(TMPDIR, { recursive: true, force: true })
+    })
+
+    async function loadSettings (content) {
+        // Need to fix the node path inside the content to ensure it can find
+        // the @flowforge/nr- modules it tries to load
+        content = `module.paths.unshift('${path.join(__dirname, '../../../node_modules')}'); ${content}`
+        const fn = path.join(TMPDIR, `${Math.random().toString(36).substring(2)}.js`)
```

Comment:
```suggestion
        const nmPath = path.normalize(path.join(__dirname, '../../../node_modules')).split(path.sep).join('/')
        content = `module.paths.unshift('${nmPath}'); ${content}`
        const fn = path.normalize(path.join(TMPDIR, `${Math.random().toString(36).substring(2)}.js`)).split(path.sep).join('/')
```

Reply:
When run in windows, the fine path contains single `\` instead of `\\` so fails to find `@flowforge/_module_` - this change will change to posix style path separators